src/v1/types/requests.rs

```diff
@@ -66,18 +71,23 @@ pub struct ListTasksParams {
     /// If the value is empty, it matches all values.
     ///
     /// It is an error if more values are supplied than keys.
-    #[cfg_attr(feature = "serde", serde(rename = "tag_value", default))]
+    #[cfg_attr(
+        feature = "serde",
+        serde(rename = "tag_value", default, skip_serializing_if = "Vec::is_empty")
+    )]
     pub tag_values: Vec<String>,
     /// The number of tasks to return in one page.
     ///
     /// Must be less than 2048. Defaults to 256.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     pub page_size: Option<u16>,
     /// The page token to retrieve the next page of results.
     ///
     /// If unspecified, returns the first page of results.
     ///
     /// The value can be found in the `next_page_token`` field of the last
     /// returned result of `list_tasks`.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     pub page_token: Option<String>,
     /// The view of the returned task(s).
```

Comment:
Similarly, `view` is listed as being optional with a default value. I wonder if there is reason to make this `Option<View>` and have the user manually `.unwrap_or_default()` it.